### PR TITLE
Table property filter

### PR DESCRIPTION
**Pull request by hicqu**
### What problem does this PR solve?

Issue Number: ref #5718

Use table property filter to speed up cdc sorter, so that CPU usage and latency can be reduced.

#### Tests 

 - Unit test

Benchmarks
#### vanilla v.s. patched, with modified configurations:
* tikv.min-ts-interval = 100ms
* cdc.owner-flush-interval = 50ms
* cdc.processor-flush-interval = 50ms

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by overvenus — COMMENTED** [review]
Rest LGTM

File: pkg/db/pebble.go [hunk 1]
```diff
@@ -94,14 +96,32 @@ func buildPebbleOption(id int, memInByte int, cfg *config.DBConfig) (pebble.Opti
...
 func OpenPebble(
 	ctx context.Context, id int, path string, memInByte int, cfg *config.DBConfig,
+) (DB, error) {
+	return doOpenPebble(ctx, id, path, memInByte, true, cfg)
+}
+
+func doOpenPebble(
+	ctx context.Context,
+	id int, path string, memInByte int,
+	withTablePropertyCollectors bool,
```
Comment (sdojjy) [inline pkg/db/pebble.go:113]: seems `withTablePropertyCollectors` is always true, why we need this flag?

**Review by sdojjy — APPROVED** [review]

File: pkg/db/pebble.go (hunk 1 above)
Comment (hicqu) [reply pkg/db/pebble.go:113]: For some tests in which keys are not valid cdc event key format.

**Review by overvenus — APPROVED** [review]

**Pull request merged**